srcpkgs/k0s/template

```diff
@@ -0,0 +1,44 @@
+# Template file for 'k0s'
+pkgname=k0s
+version=0.9.1
+revision=1
+build_style=go
+go_import_path=github.com/k0sproject/k0s
+hostmakedepends="which golangci-lint"
+depends="kubernetes>1.20_1 containerd>=1.3.9_1"
```

Comment:
I don't think it's necessary to specify these versions, unless you believe people are likely to be holding the packages manually?

Reply:
I'd really like to depend on the same versions k0s bundles (I deactiveted bundling for the void packages)

Reply:
Actually, I even needed to depend on 1.3.9 instead of the bundled 1.4 release, but it seems to work with the older release as well.